spec/fabricators/application_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:application, from: Doorkeeper::Application) do
+  name 'iOS client'
+  redirect_uri ''
+  scopes ''
```

Comment:
As we are not setting any default value, I think we can remove it
```suggestion

```

Reply:
working! :)